roles/chaos-beav/templates/kill-container.sh.j2

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
```

Comment:
do you think we could avoid having this script as a template? you're using `{% raw %}` for most of it, and storing it as a plain shell script would be easier to test/debug in the future.